frontends/web/src/routes/account/send/send.tsx

```diff
@@ -14,35 +14,94 @@
  * limitations under the License.
  */
 
-import { Component, h } from 'preact';
+import { Component, h, RenderableProps } from 'preact';
 import { route } from 'preact-router';
-import { translate } from 'react-i18next';
-import { isBitcoinBased } from '../utils';
-import { apiGet, apiPost } from '../../../utils/request';
-import { apiWebsocket } from '../../../utils/websocket';
-import { debug } from '../../../utils/env';
+import reject from '../../../assets/icons/cancel.svg';
+import approve from '../../../assets/icons/checked.svg';
+import { alertUser } from '../../../components/alert/Alert';
+import { Balance, BalanceInterface } from '../../../components/balance/balance';
 import { Button, ButtonLink, Checkbox, Input } from '../../../components/forms';
-import { Guide } from '../../../components/guide/guide';
 import { Entry } from '../../../components/guide/entry';
-import { store as fiat } from '../../../components/rates/rates';
-import { alertUser } from '../../../components/alert/Alert';
+import { Guide } from '../../../components/guide/guide';
 import { Header } from '../../../components/layout';
+import { Fiat, store as fiat } from '../../../components/rates/rates';
 import Status from '../../../components/status/status';
 import WaitDialog from '../../../components/wait-dialog/wait-dialog';
-import { Balance } from '../../../components/balance/balance';
+import { translate, TranslateProps } from '../../../decorators/translate';
+import { debug } from '../../../utils/env';
+import { apiGet, apiPost } from '../../../utils/request';
+import { apiWebsocket } from '../../../utils/websocket';
+import { isBitcoinBased } from '../utils';
 import FeeTargets from './feetargets';
-import UTXOs from './utxos';
-import approve from '../../../assets/icons/checked.svg';
-import reject from '../../../assets/icons/cancel.svg';
 import * as style from './send.css';
+import { Props as UTXOsProps, SelectedUTXO, UTXOs } from './utxos';
```

Comment:
Why do you import `Props` (instead of `UTXOsProps `)? It is declared as `export type Props = UTXOsProps & TranslateProps` and we don't need the `t` here.